examples/with-typescript/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "dev": "concurrently 'tsc --watch' 'next'"
+  },
+  "dependencies": {
+    "next": "^1.2.3"
```

Comment:
This doesn't seem to depend on a specific version of next. Can you use `latest` here?